packages/sync/src/config/DefaultConfig.ts

```diff
@@ -1,6 +1,10 @@
 import { INSTANCE, ServiceConfiguration } from "@aerogear/core";
 import { ConfigError } from "./ConfigError";
 import { DataSyncConfig } from "./DataSyncConfig";
+import {NetworkStatus} from "../offline/NetworkStatus";
```

Comment:
if you use visual studio code or webstorm please consider using editorconfig plugin to support formatting.